msal/build.gradle

```diff
@@ -179,7 +179,7 @@ task sourcesJar(type: Jar) {
     destinationDirectory = reporting.file("$project.buildDir/outputs/jar/")
 }
 
-String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "0.0.+"
+String commonVersion = project.hasProperty("distCommonVersion") ? project.distCommonVersion : "6.0.0-RC1"
```

Comment:
@iamgusain not sure if this should be updated.

Reply:
Yes, this should be the specific common version for release/* branches.